src/discussions/discussions-home/DiscussionsHome.scss

```diff
@@ -0,0 +1,3 @@
+.nav-shadow {
+  box-shadow: 0px 2px 4px rgb(0 0 0 / 15%), 0px 2px 8px rgb(0 0 0 / 15%);
+}
```

Comment:
should we not made that fix in paragon?

Reply:
.shadow |   | box-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.3) !important;
-- | -- | --
.shadow-lg |   | box-shadow: 0 0.25rem 0.5rem rgba(0, 0, 0, 0.3) !important;
.shadow-none |   | box-shadow: none !important;
.shadow-sm |   | box-shadow: 0 0.0625rem 0.125rem rgba(0, 0, 0, 0.2) !important;

These classes are available in paragon , Please verify if any is related probably `.shadow-sm`

https://paragon-openedx.netlify.app/foundations/css-utilities/

Reply:
Discussion Figma designs are using old paragon styles which are affected because the latest paragon changes. The above classes have been changed in the latest paragon and do not match with our Figma design. That's why I created a custom CSS class to meet Figma's requirement.

Reply:
I feel, I may be wrong but we should go with the updated thing and ask Jonathan on this one what he suggests.

Reply:
According to Jonathan, He is using shadow elevation level 2 in this design. Shadow elevation effects exist in paragon as SCSS mixin `pgn-box-shadow`. This mixin is available in only paragon but not available as a class to use across platforms. It will be implemented in paragon's future milestones. So, we decided to add a custom style here and replace it with paragon class after its paragon implementation.